### PR TITLE
pipeline: outputs: dash0: Added Dash0 documentation

### PR DESCRIPTION
**Pull request by CodingFabian**
Added documentation on how to send logs to dash0 based on the OpenTelemetry output plugin.

### CODE REVIEW

File: pipeline/outputs/dash0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| `tls.verify`               | TLS verification. | `on` |
+
+## Getting started
+
+To get started with sending logs to Dash0:
+
+1. Get an [Auth Token](https://www.dash0.com/documentation/dash0/key-concepts/auth-tokens) from **Settings** > **Auth Tokens**.
+1. In your main Fluent Bit configuration file, append the following `Output` section:
+
+   ```text
```
Comment (patrick-stephens) [inline pipeline/outputs/dash0.md:29]: Indentation and missing closing ``` section

We're also trying to make sure we include YAML as well as classic config so if you can add that?

File: pipeline/outputs/dash0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   [OUTPUT]
+      Name:         opentelemetry
```
Comment (patrick-stephens) [inline pipeline/outputs/dash0.md:31]: The colons `:` are incorrect here, that's the YAML format :)

File: pipeline/outputs/dash0.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      Match:        *
+      Host:         ingress.eu-west-1.aws.dash0.com
+      Port:         443
+      Header:       Authorization Bearer auth_vdOxPqcvSlBkhVQV95wU9TGXh2Fdjliq
+      Metrics_uri:  /v1/metrics
+      Logs_uri:     /v1/logs
+      Traces_uri:   /v1/traces
+      Tls:          On
+      Tls.verify:   On
+
```
Comment (patrick-stephens) [inline pipeline/outputs/dash0.md:41]: Missing closing dashes to end the text box.

File: pipeline/outputs/dash0.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+| Key                        | Description | Default |
+| -------------------------- | ----------- | ------- |
+| `header`                   | The specific header for bearer authorization, where {your-Auth-token-here} is your Dash0 Auth Token. | Authorization Bearer {your-Auth-token-here} |
+| `host`                     | Your Dash0 ingress endpoint. | `ingress.eu-west-1.aws.dash0.com` |
+| `port`                     | TCP port of your Dash0 ingress endpoint. | `443` |
+| `metrics_uri`              | Specify an optional HTTP URI for the target web server listening for metrics | `/v1/metrics` |
+| `logs_uri`                 | Specify an optional HTTP URI for the target web server listening for logs | `/v1/logs` |
+| `traces_uri`               | Specify an optional HTTP URI for the target web server listening for traces | `/v1/traces`  |
+| `tls`                      | Specify to use TLS. | `on` |
```
Comment (patrick-stephens) [inline pipeline/outputs/dash0.md:19]: We can probably skip this and just reference the transport security section as these are common to all.

**Review by patrick-stephens — COMMENTED** [review]
Looks fine from technical perspective but will allow docs (@fluent/chronosphere-tech-writers ) to confirm the writing perspective.

File: pipeline/outputs/dash0.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+---
+description: Send logs to Dash0
+---
+
+# Dash0
+
+Stream logs to [Dash0](https://www.dash0.com) by utilizing the **opentelemetry** plugin to send data to the Dash0 log ingress.
```
Comment (esmerel) [inline pipeline/outputs/dash0.md:7]:
```suggestion
Stream logs to [Dash0](https://www.dash0.com) by utilizing the OpenTelemetry plugin to send data to the Dash0 log ingress.
```

It'd be nice to crosslink to the plugin, but not required.

File: pipeline/outputs/dash0.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+| `header`                   | The specific header for bearer authorization, where {your-Auth-token-here} is your Dash0 Auth Token. | Authorization Bearer {your-Auth-token-here} |
+| `host`                     | Your Dash0 ingress endpoint. | `ingress.eu-west-1.aws.dash0.com` |
+| `port`                     | TCP port of your Dash0 ingress endpoint. | `443` |
+| `metrics_uri`              | Specify an optional HTTP URI for the target web server listening for metrics | `/v1/metrics` |
+| `logs_uri`                 | Specify an optional HTTP URI for the target web server listening for logs | `/v1/logs` |
+| `traces_uri`               | Specify an optional HTTP URI for the target web server listening for traces | `/v1/traces`  |
+
+### TLS / SSL
+
+The opentelemetry output plugin supports TLS/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](../../administration/transport-security.md) section.
```
Comment (esmerel) [inline pipeline/outputs/dash0.md:22]:
```suggestion
The OpenTelemetry output plugin supports TLS/SSL. For more details about the properties available and general configuration, see  [TLS/SSL](../../administration/transport-security.md).
```

**Review by esmerel — APPROVED** [review]
A couple of suggestions so we're using proper nouns properly =)

I'll go ahead and approve as well,  if you'd update those two changes that'd be great.

And thanks for updating the spelling dictionary!

File: pipeline/outputs/dash0.md (hunk 6 above)
Comment (patrick-stephens) [reply pipeline/outputs/dash0.md:22]: Let's break any new sentences into new lines as well, makes it easier to review and comment on specific sentences then.

Comment (CodingFabian) [reply pipeline/outputs/dash0.md:22]: Its how I copied it from another place. You want me to edit it there?

**Comment by CodingFabian** [comment]
@esmerel I have changed all references to your proposed version in:
https://github.com/fluent/fluent-bit-docs/pull/1532/commits/9a5befacf6e4deb97ed7e3c8b3a636f893d549f1
left the other commit just for dash0

File: pipeline/outputs/dash0.md (hunk 6 above)
Comment (esmerel) [reply pipeline/outputs/dash0.md]: I wouldn't stress about editing it in the place you copied from. It'll get cleaned up eventually =)

Comment (CodingFabian) [reply pipeline/outputs/dash0.md]: I preferred to do it now. it feels strange to make the only copy of a sentence that differs from the others that were copied ten times.

**Review by patrick-stephens — CHANGES_REQUESTED** [review]
Ah, one thing left to do is add your new page to https://github.com/fluent/fluent-bit-docs/blob/master/SUMMARY.md so it appears @CodingFabian

File: SUMMARY.md [hunk 7]
```diff
@@ -178,6 +178,7 @@
   * [Azure Log Analytics](pipeline/outputs/azure.md)
   * [Azure Logs Ingestion API](pipeline/outputs/azure_logs_ingestion.md)
   * [Counter](pipeline/outputs/counter.md)
+  * [Dash0](pipeline/outputs/dash0.md)
```
Comment (CodingFabian) [inline SUMMARY.md:181]: is not this one @patrick-stephens ?

Comment (patrick-stephens) [reply SUMMARY.md:181]: Weird, I checked a few times to confirm and missed it :)

**Pull request merged**